docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md

```diff
@@ -0,0 +1,141 @@
+# How to integrate Autoware with your vehicle
+
+## 1. Prepare your real vehicle hardware
+
+Prerequisites for the vehicle:
+
+- An onboard computer that satisfies the Autoware Installation prerequisites (see [here](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/#prerequisites))
+- The following devices attached
+  - Drive-by-wire interface
+  - LiDAR
+  - Optional: Inertial measurement unit
+  - Optional: Camera
+  - Optional: GNSS
+
+## 2. Create maps
+
+You need both a pointcloud map and a vector map in order to use Autoware.
+
+### Create a pointcloud map
+
+Use third-party tools such as a LiDAR-based SLAM (simultaneous localization and mapping) package to create a pointcloud map in the `.pcd` format.
+
+### Create vector map
+
+Use third-party tools such as [TIER IV's Vector Map Builder](https://tools.tier4.jp/) to create a Lanelet2 format `.osm` file.
+
+## 3. Create your meta-repository
+
+Create your Autoware repository.
+One easy way is to fork [autowarefoundation/autoware](https://github.com/autowarefoundation/autoware) and clone it ([how to fork repository on github](https://docs.github.com/en/get-started/quickstart/fork-a-repo)).
+
+```bash
+git clone https://github.com/YOUR_NAME/autoware.YOURS.git
+```
+
+## 4. Create the description packages of your vehicle
+
+Next, you need to create description packages that define the vehicle and sensor configuration of your vehicle.
+Once it is done, you can launch your vehicle model by specifying `vehicle_model:=YOUR_VEHICLE` `sensor_model:=SAMPLE_SENSOR_KIT` in the autoware launchers.
+
+Create the following two packages:
+
+- YOUR_VEHICLE_launch (see [here](https://github.com/autowarefoundation/sample_vehicle_launch) for example)
+- YOUR_SENSOR_KIT_launch (see [here](https://github.com/autowarefoundation/sample_sensor_kit_launch) for example)
+
+Once created, you need to update the `autoware.repos` file of your cloned Autoware repository to refer to these two description packages.
+
+### Adapt YOUR_VEHICLE_launch for autoware launching system
+
+#### At YOUR_VEHICLE_description
+
+Define URDF and parameters in the vehicle description package (refer to the [sample vehicle description package](https://github.com/autowarefoundation/sample_vehicle_launch/tree/main/sample_vehicle_description) for an example).
+
+#### At YOUR_VEHICLE_launch
+
+Create a launch file (refer to the [sample vehicle launch package](https://github.com/autowarefoundation/sample_vehicle_launch/tree/main/sample_vehicle_launch) for example).
+If you have multiple vehicles with similar hardware setup, you can specify `vehicle_id` to distinguish them.
```

Comment:
Are there any reasons you used `similar`?

```suggestion
If you have multiple vehicles with the same hardware setup, you can specify `vehicle_id` to distinguish them.
```

Reply:
Not particularly. Thanks for pointing it out.